### DISCUSSION TITLE
gofiber update

### DISCUSSION THREAD

**Discussion by ionutdirzubitpanda** [roadrunner-server/roadrunner#1878] [Q&A] [answered]
Hello,

Apparently there is a problem with the **gofiber** package on the last tag, the one of 2023.3.11.
The package has version v2.52.0 which seems to be marked as a vulnerability here https://avd.aquasec.com/nvd/2024/cve-2024-25124, the fix seems to be in version 2.52.1
I notice that the update is done on the current version in the master, can a new tag be made with the current version?

Thank you!

**Comment by rustatian** [comment, marked as answer]
Hey @ionutdirzubitpanda 👋 Thanks for noting that 👍 
We're not using the affected middleware in the RR (nor exposing configuration for it), so, there is no need to rush. The new version will be released on Thursday.
